tile.yml.erb

```diff
@@ -66,20 +66,16 @@ forms:
   properties:
   - name: rlp_address
     type: string
-    label: Reverse Log Proxy IP address and port
-  - name: rlp_cacert
-    type: text
-    label: CA certificate of the Reverse Log Proxy
-  - name: rlp_cert
-    type: text
-    label: The TLS certificate for the Reverse Log Proxy, found in the Adapter Rlp Tls section of your PAS tile credentials section
-  - name: rlp_key
-    type:  text
-    label: The TLS key for the Reverse Log Proxy, found in the same place as the cert
-  - name: firehose_endpoint
-    type: string
-    label: Cloud Foundry API Endpoint
-    default: (( $runtime.system_api_url ))
+    label: Reverse Log Proxy host and port
+    description: Address and port of the reverse log proxy. The default value works if BOSH DNS is enabled. Otherwise, set this to the IP address of a reverse log proxy VM.
+    default: loggregator-trafficcontroller.service.cf.internal:8082
```

Comment:
I wonder if it's possible to discover this via a [BOSH link](https://bosh.io/docs/links/)?

Reply:
Should be, I'll investigate tomorrow and update this PR